server/src/main/scala/actors/ShiftsActorBase.scala

```diff
@@ -24,20 +25,16 @@ case object ClearState
 
 case object GetFeedStatuses
 
-case class PointInTimeQuery(pointInTime: MillisSinceEpoch, query: DateRangeLike)
+case class PointInTimeQuery(pointInTime: MillisSinceEpoch, query: DateRangeLike) extends PortStateRequest
```

Comment:
check if we still need the PortStateRequest trait

Reply:
still used